src/components/Button/Button.tsx

```diff
@@ -156,6 +164,7 @@ class Button extends UIComponent<Extendable<ButtonProps>, ButtonState> {
         disabled={disabled}
         onClick={this.handleClick}
         onFocus={this.handleFocus}
+        type={type}
```

Comment:
if we decide to remove type from handled props don't forget to remove it from here as well